src/libraries/System.Runtime.InteropServices.WindowsRuntime/src/System.Runtime.InteropServices.WindowsRuntime.csproj

```diff
@@ -4,6 +4,9 @@
     <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <GenFacadesIgnoreMissingTypes Condition="'$(RuntimeFlavor)' == 'Mono'">true</GenFacadesIgnoreMissingTypes>
+    <GenFacadesIgnoreMissingType Condition="'$(RuntimeFlavor)' == 'Mono'">true</GenFacadesIgnoreMissingType>
```

Comment:
Other projects use `GenFacadesIgnoreMissingType` as an item group with all the type names and not a property. This doesn't look as the correct usage.

Reply:
Good call. I will go back through here and get the full list. Also need to file a follow up issue for Mono to rationalize the gap.